openc3/templates/command_validator/command_validator.py

```diff
@@ -0,0 +1,45 @@
+from openc3.packets.command_validator import CommandValidator
+# Using the OpenC3 API requires the following imports:
+# from openc3.api import wait_check
+
+# Custom command validator class
+# See https://docs.openc3.com/docs/configuration/command
+class <%= validator_class %>(CommandValidator):
+    def __init__(self, *args):
+        super().__init__()
+        self.args = args
+
+    # Called before a command is sent
+    # @param command [dict] The command dictionary containing all the command details
+    # @return [list] First element is True/False/None for success/failure/unknown,
+    #   second element is an optional message string
+    def pre_check(self, command):
+        # Add your pre-command validation logic here
+        # Example:
+        # target_name = command['target_name']
+        # command_name = command['cmd_name']
+        # params = command['cmd_params']
+        #
+        # if some_condition:
+        #     return [False, "Command validation failed: reason"]
+
```

Comment:
Add an example of using tlm to grab a value and then check it later. So in the pre_check it would be: `self.count = tlm("TARGET PACKET COUNT")`